client/src/components/Authentication/protected-route.tsx

```diff
@@ -1,25 +1,32 @@
 /*
 This component takes a component in as a prop,
-and only retrns it if one is logged in.
+and only returns it if one is logged in and an admin.
 */
```

Comment:
Could you move this comment to replace the one associated with the `ProtectedRoute` component below on line 9-10